src/platform/apollolake/include/platform/platform.h

```diff
@@ -150,6 +150,9 @@ struct sof;
 /* minimal SSP port stop delay in cycles */
 #define PLATFORM_SSP_STOP_DELAY	2400
 
+/* timer driven scheduling offset in microseconds */
```

Comment:
if this is an offset from trigger start ? can we state this and any other assumptions about size.